### ISSUE TITLE
Coefficient of restitution

### ISSUE THREAD

**Issue by sayednaseel** [biomechanics-course/hw00#1]
Should the coefficient of restitution be set to 0.5 instead of -0.5? Within the code, the coefficient of restitution is set to 0.9. 
```
e = 0.9;    % coefficient of restitution for bouncing
y0(2) = -e*y(nt,2);    % After each bounce, take the ending velocity
                                  % of the previous segment, and reverse it and
                                 % multiply by the coefficient of restitution.
```
A coefficient of restitution of -0.5 would make the ball continue moving into the floor after the first collision.

**Comment by anthony93c** [comment]
Same question! I did it with 0.5 to simulate a bounce.

**Comment by artkuo** [comment]
Yes, coefficient of restitution should be positive. Can someone file a pull request to fix the README?

**Comment by sayednaseel** [comment]
I have changed the README.md file in the git aware folder on my laptop. I then created a new branch called SayedHW00 in Git Bash. I added and committed the changes to this branch. But when I try to push the changes to the online repository, I get this error: 
```
remote: Permission to biomechanics-course/hw00.git denied to sayednaseel.        fatal: unable to access 'https://github.com/biomechanics-course/hw00.git/': The  requested URL returned error: 403
--
```
I switched to the Github Desktop, where it states that my current branch SayedHW00 has not been published to the remote yet. And that I have to push the changes to remote, before I can setup a pull request. When i click on publish branch, it says "Authentication failed. You do not have permission to access the repository". I have signed in using my Github account on Github Desktop. 

Does anyone else face these issues? Were you able to create a branch to the master branch here?

**Comment by artkuo** [comment]
Thanks for trying that. For Markdown documents, it turns out there's a super easy way to file pull requests. On the Readme header, there's a pencil icon that will allow you to edit. One of the choices is to edit directly if you have permission, and the other is to create a branch and edit that. Once you complete the branch, it will allow you to file a pull request. Pull request sends an automated message to me, so I can review and accept, or start a discussion on it. 

Of course, I can also edit and fix the error myself. But the reason to do a pull request is to demonstrate to everyone (and get some practice) in seeing how GitHub facilitates collaboration.

**Comment by timvanderzee** [comment]
> Yes, coefficient of restitution should be positive. Can someone file a pull request to fix the README?

I don't understand that Art. The matlab comments say: "After each bounce, take the ending velocity of the previous segment, and reverse it and multiply by the coefficient of restitution." This to me makes sense: if you have a downward (i.e. negative) velocity, you will have an upward (e.g. positive) velocity after collision with the ground. I'd say the minus sine before e was appropriate after all..
